test/spec/features/complex-preview/ComplexPreviewSpec.js

```diff
@@ -0,0 +1,181 @@
+import {
+  bootstrapDiagram,
+  inject
+} from 'test/TestHelper';
+
+import complexPreviewModule from 'lib/features/complex-preview';
+import modelingModule from 'lib/features/modeling';
+import rendererModule from '../preview-support/renderer';
+
+import {
+  query as domQuery,
+  queryAll as domQueryAll
+} from 'min-dom';
+
+var testModules = [
+  complexPreviewModule,
+  modelingModule,
+  rendererModule
+];
+
+
+describe('features/complex-preview - ComplexPreviewSpec', function() {
```

Comment:
```suggestion
describe('features/complex-preview', function() {
```

Reply:
I copied this from another file so we'd probably want to change it in all other files as well then.